.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,7 +7,7 @@ about: Create a report to help us improve
 <!--
 READ THIS FIRST!
 
-If you have a usage question, please ask us on either Stack Overflow (https://stackoverflow.com/questions/tagged/jq) or in the #jq channel (https://web.libera.chat/#jq) on Libera.Chat (https://libera.chat/).
+If you have a usage question, please ask us on either Stack Overflow (https://stackoverflow.com/questions/tagged/jq) or in the #general channel on [Discord](https://discord.gg/yg6yjNmgAC).
```

Comment:
You forgot to mention the IRC channel here

Reply:
Fixed